docs/samples/Microsoft.ML.Samples/Dynamic/SsaSpikeDetectorTransform.cs

```diff
@@ -66,30 +75,135 @@ public static void SsaSpikeDetectorTransform()
             var predictionColumn = transformedData.AsEnumerable<SsaSpikePrediction>(ml, reuseRowObject: false);
 
             Console.WriteLine($"{outputColumnName} column obtained post-transformation.");
-            Console.WriteLine("Alert\tScore\tP-Value");
+            Console.WriteLine("Data\tAlert\tScore\tP-Value");
+            int k = 0;
             foreach (var prediction in predictionColumn)
-                Console.WriteLine("{0}\t{1:0.00}\t{2:0.00}", prediction.Prediction[0], prediction.Prediction[1], prediction.Prediction[2]);
+                Console.WriteLine("{0}\t{1}\t{2:0.00}\t{3:0.00}", data[k++].Value, prediction.Prediction[0], prediction.Prediction[1], prediction.Prediction[2]);
             Console.WriteLine("");
 
             // Prediction column obtained post-transformation.
-            // Alert   Score   P-Value
-            // 0       0.00    0.50
-            // 0       0.00    0.50
-            // 0       0.00    0.50
-            // 0       0.00    0.50
-            // 0       0.00    0.50
-            // 0       0.00    0.50
-            // 0       0.00    0.50
-            // 0       0.00    0.50
-            // 1       5.00    0.00   <-- alert is on, predicted spike
-            // 0       -2.50   0.09
-            // 0       -2.50   0.22
-            // 0       0.00    0.47
-            // 0       0.00    0.47
-            // 0       0.00    0.26
-            // 0       0.00    0.38
-            // 0       0.00    0.50
-            // 0       0.00    0.50
+            // Data    Alert   Score   P-Value
+            // 0       0     - 2.53    0.50
+            // 1       0     - 0.01    0.01
+            // 2       0       0.76    0.14
+            // 3       0       0.69    0.28
+            // 4       0       1.44    0.18
+            // 0       0     - 1.84    0.17
+            // 1       0       0.22    0.44
+            // 2       0       0.20    0.45
+            // 3       0       0.16    0.47
+            // 4       0       1.33    0.18
+            // 0       0     - 1.79    0.07
+            // 1       0       0.16    0.50
+            // 2       0       0.09    0.50
+            // 3       0       0.08    0.45
+            // 4       0       1.31    0.12
+            // 100     1      98.21    0.00   <-- alert is on, predicted spike
+            // 0       0    - 13.83    0.29
+            // 1       0     - 1.74    0.44
+            // 2       0     - 0.47    0.46
```

Comment:
doe it make sense to limit the preview/data to something smaller,maybe like 7  rows ?